io.playnux.calculator.yml

```diff
@@ -0,0 +1,26 @@
+---
+
+app-id: io.playnux.calculator
+runtime: io.elementary.Platform
```

Comment:
this runtime doesn't exist.

Reply:
no it doesn't on flathub.

Reply:
I have no idea. There is an elementary base app but it was my understanding that it was too old.

Maybe you should convince the Elementary people to publish their SDK on Flathub.